### DISCUSSION TITLE
RBAC Problems with Pipelines from Jyupiter notesboosk

### DISCUSSION THREAD

**Discussion by Syndlex** [kubeflow/pipelines#10773] [General]
Hey,

We're trying to create pipelines from within a Notebook started by Kubeflow.

Currently, we're encountering this error in the Pipeline Logs:

 > Unauthenticated: Request header error: there is no user identity header.: Request header error: there is no user identity header.
    Unauthenticated: Failed to check authorization. User identity is empty in the request header

We can upload pipelines normally, but as soon as we want to access something else, we get this error. Can someone explain where a request is routed and what access needs to be configured?

We've defined a default pod, and the access token is within the Notebook. I don't understand where the information is missing.
 The whole problem is connected,to this issue: https://github.com/kubeflow/pipelines/issues/4440

**Comment by rimolive** [comment]
We need more information about your environment, so let me ask the first questions:
* Kubeflow Version?
* KFP backend and SDK versions?

**Reply by Syndlex** [reply]
Hey, 

We are using the manifests in version 1.8.1

Pipeline image is 2.0.5
Ask version we tried: 1.8.22 and 2.7.0

**Reply by rimolive** [reply]
So with SDK 1.8.22 and 2.7.0 you get the same result. Did I get it right?
